packages/database/src/schema/models/convos.ts

```diff
@@ -0,0 +1,35 @@
+import { InferSelectModel, relations, sql } from 'drizzle-orm'
+import { bigint, bigserial, index, jsonb, uuid } from 'drizzle-orm/pg-core'
+
+import { latitudeSchema } from '../db-schema'
+import { timestamps } from '../schemaHelpers'
+import { PromptVersion, promptVersions } from './promptVersions'
+
+export const convos = latitudeSchema.table(
+  'convos',
+  {
+    id: bigserial('id', { mode: 'bigint' }).notNull().primaryKey(),
+    uuid: uuid('uuid')
+      .notNull()
+      .unique()
+      .default(sql`gen_random_uuid()`),
+    content: jsonb('content'),
+    promptVersionId: bigint('prompt_version_id', { mode: 'bigint' })
+      .notNull()
+      .references(() => promptVersions.id, { onDelete: 'cascade' }),
```

Comment:
not sure if we should attach conversations to versions or snapshots. Why you choose versions?

Reply:
Because you can always trigger a convo by referencing a past commit, and so it makes sense to target the prompt version for that commit.

Reply:
But the `prompt_snapshots` is who holds the relation with commits

Reply:
oh I didn't realize. I think the model is wrong? If I ask for a prompt in a particular commit I'm gonna build the snapshot in memory, use it, and remove it once completed. In our convos we said prompt_snapshot were only the snapshot of the latest commit.

Reply:
You're right. Is missing `commitId`